element-connector/src/main/java/org/eclipse/californium/elements/RawData.java

```diff
@@ -330,4 +330,16 @@ public boolean isSecure() {
 		return (correlationContext != null &&
 				correlationContext.get(DtlsCorrelationContext.KEY_SESSION_ID) != null);
 	}
+	
+	/**
+	 * Callback, when context gets available. Used on sending an message.
+	 * 
+	 * @param context established context to be forwarded to the callback.
+	 */
+	public void onContextEstablished(CorrelationContext context) {
```

Comment:
It may be usefull, if the correlation conntext is not yet available, and the aquire it later has some overhead. I just want to reduce the always same "check & call".